### PR TITLE
changelog: update changes

### PR DESCRIPTION
**Pull request by fanminshi**
[skip ci]

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hasbro17 @hongchaodeng

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
+- Added tag to initContainer to enable offline deploys
+- Enabled configurable backup timeout in backup operator 
+
 ### Changed
 
+- Set 30s default request timeout for kube client
+
 ### Removed
 
+- Remove self-hosted code
```
Comment (hasbro17) [inline CHANGELOG.md]: *Removed

**Comment by hasbro17** [comment]
LGTM after nit.

**Comment by fanminshi** [comment]
fixed.

**Pull request merged**